src/docs/releases/1.8.0.md

```diff
@@ -12,6 +12,10 @@ The interface `OrchardCore.Documents.IDocumentSerialiser` has been renamed to `O
 
 This release removes support for `net6.0` and `net7.0`. Only `net8.0` is supported.
 
+### Email
+
+The `MailMessage.Body` now returns `MailMessageBody` instead of `string` to support both plain text & HTML formats.
```

Comment:
This isn't needed in the 1.8 release notes.

Reply:
It was planned for `1.8` :)